.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/220860889-02c14711-5f58-4994-997f-1ea03743ca3e.png)

- [ ] It would be nice if you remove your **Live Demo** section since you don't have any live link for your application. Kindly remove it to make your readme file descriptive, professional, and with the actual information of your application. :blush:

Reply:
Hi @HAMMAS-SALEEM, thanks very much for your time. In the past (in a previous project), I had removed the section but I was notified that the "Live Demo" section is mandatory and that I had to add the "Coming soon" part.

As you can see in the [Demo template](https://github.com/microverseinc/readme-template) the only optional section is the FAQ section.

Please, can you give more clear instructions?

Reply:
Hey, @anthonymr I hope you are doing well :blush: I have talked with my Senior and he said that students need to remove the **Live Demo** Section unless they have a working live link for their application :+1:

Here is the instruction of my Senior :point_down:
![image](https://user-images.githubusercontent.com/93039359/220864086-356bb39b-f46a-49d3-9cb5-eb664a334dd8.png)

Reply:
Hi, @HAMMAS-SALEEM , thank you for the research. Now I know I'm safe deleting this section in the future.

Again, thank you very much.